cypress_test/integration_tests/common/helper.js

```diff
@@ -375,11 +375,20 @@ function documentChecks(skipInitializedCheck = false) {
 						});
 				});
 		});
+
+		// In Writer wait for styles to appear in notebookbar
+		doIfOnDesktop(() => {
+			doIfInWriter(() => {
+				cy.cGet('#stylesview.notebookbar .icon-view-item-container img')
+					.should('exist');
+			});
+		});
+
 		// Check also that the inputbar is drawn in Calc.
-		doIfInCalc(function() {
-			cy.cframe().get('#sc_input_window.formulabar').should('exist');
-			cy.cframe().get('#pos_window-input.addressInput').should('exist');
-			cy.cframe().get('#pos_window-input.addressInput').should('not.be.empty');
+		doIfInCalc(() => {
```

Comment:
Just curious: is () => {} a syntax sugar only, or is it actually different from `function() { ... }`?

Reply:
Here it doesn't matter.

Most important is difference in `this`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#cannot_be_used_as_methods

In most cases you want to use wrapping object - so you don't need to `.bind(this)` or use `var that = this` everywhere :)